pkg/controller/migmigration/task.go

```diff
@@ -275,3 +406,85 @@ func (t *Task) addErrors(errors []string) {
 		t.Errors = append(t.Errors, error)
 	}
 }
+
+// Remove stage pods on source and destination cluster
+func (t *Task) removeStagePods() error {
```

Comment:
I wonder if any rate limiting on this type of action will be needed. I could see us sending a bunch of deletes on pods that are in the process of deleting.

Reply:
Good point. I suppose we could add a check to see if the pod is in a `Terminating` state and don't send the delete?